ember-toucan-core/src/components/form/file-input-field.hbs

```diff
@@ -0,0 +1,39 @@
+<div
+  class="flex flex-col {{if @isDisabled 'text-disabled'}}"
+  data-root-field={{if @rootTestSelector @rootTestSelector}}
+>
+  <Form::Field as |field|>
+    <field.Label for={{field.id}}>
```

Comment:
I just noticed by playing with this in https://da42f841.ember-toucan-core.pages.dev/docs/components/file-input-field, that having the label wrap the whole control and hint, it causes the whole block with a with of 100% to be clickable. For example, if you click _right of_ the upload button (i.e. not hitting the button!), then the file dialog will pop up nevertheless! 

I don't think that's what we want, right?

What was the reason for wrapping the label, instead of the usual approach we have elsewhere?

Reply:
Hm, not sure about this. I still feel this should get solved in some way, but this can be done in a follup-up PR I think. This one already has so many comments! 😅 

Maybe we can find examples in the wild for how others have solved this?

Reply:
I have an idea about this, will handle in a followup: https://github.com/CrowdStrike/ember-toucan-core/issues/86